client/src/assets/scss/theme.scss

```diff
@@ -46,25 +46,16 @@ $BCgovAGold2: #FFF8EF;
 $BCgovAGold3: #FFE0BC;
 $BCgovAGold4: #FFD4A2;
 
-$border: #899EB2;
-
-// Link
-$link: #1669bb;
-
-// Approved
-$approved: #1a9031;
+// Font colors
+$dk-text: $gray9;
+$text: $gray7;
 
-// Caution
-$caution: #bf4803;
+// Some popular colors
+$app-blue: #1669bb; // same as $link
+$app-red: #d3272c; // same as $error
+$app-green: #2e8540; // not the same as $approved !?
 
-// Error Colors
-$BCgovInputError: #ff5252;
-$error: #d3272c;
-
-// Form Inputs
-$BCgovInputBG: #ffffff;
-
-// Font Colors
-$BCgovFontColorInverted: #ffffff;
-$dk-text: #212529;
-$text: #495057;
+$approved: #1a9031; // not the same as $app-green !?
+$error: #d3272c; // see also $app-red
+$border: #899EB2;
+$link: #1669bb; // see also $app-blue
```

Comment:
There is some duplication here. Any volunteers to clean it up?

Also I see there are 2 greens. Tracey specified "app-green" for me. Did she also specify "approved" green? If so then we should ask her for a single value.

Reply:
I checked with Tracey and she provided just 1 green :)